### PR TITLE
add r-sensemakr and r-srvyr packages

### PR DESCRIPTION
**Pull request by vitodelgrec**
Checklist
- [ x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ x] Source is from official source.
- [ x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ x] If static libraries are linked in, the license of the static library is packaged.
- [ x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ x] Build number is 0.
- [ x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by vitodelgrec** [comment]
@conda-forge/help-r, ready for review!

File: recipes/r-sensemakr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('sensemakr')"           # [not win]
+    - "\"%R%\" -e \"library('sensemakr')\""  # [win]
+
+about:
+  home: https://github.com/carloscinelli/sensemakr
+  license: GPL-2.0-or-later
```
Comment (mfansler) [inline recipes/r-sensemakr/meta.yaml:39]: License reads `GPL-3`, so

```suggestion
  license: GPL-3.0-only
```

File: recipes/r-sensemakr/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    variables in regression models, as discussed in Cinelli, C. and Hazlett, C. (2020),
+    "Making Sense of Sensitivity: Extending Omitted Variable Bias." Journal of the Royal
+    Statistical Society, Series B (Statistical Methodology) <doi:10.1111/rssb.12348>.'
+  license_family: GPL3
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'
+
+extra:
+  recipe-maintainers:
+    - vitodelgrec
```
Comment (mfansler) [inline recipes/r-sensemakr/meta.yaml:51]: We suggest including the Conda Forge R Team:

```suggestion
    - conda-forge/r
    - vitodelgrec
```

File: recipes/r-srvyr/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    style when calculating summary statistics on survey data using functions from the
+    'survey' package.
+  license_family: GPL3
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'
+
+extra:
+  recipe-maintainers:
+    - vitodelgrec
```
Comment (mfansler) [inline recipes/r-srvyr/meta.yaml:66]: We suggest including the Conda Forge R Team:

```suggestion
    - conda-forge/r
    - vitodelgrec
```

File: recipes/r-srvyr/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    - r-tidyselect
+    - r-vctrs >=0.3.0
+
+test:
+  commands:
+    - $R -e "library('srvyr')"           # [not win]
+    - "\"%R%\" -e \"library('srvyr')\""  # [win]
+
+about:
+  home: http://gdfe.co/srvyr/, https://github.com/gergness/srvyr/
```
Comment (mfansler) [inline recipes/r-srvyr/meta.yaml:54]: Split URLs:

```suggestion
  home: http://gdfe.co/srvyr/
  dev_url: https://github.com/gergness/srvyr/
```

**Review by mfansler — CHANGES_REQUESTED** [review]
Thanks for submitting these! ❤️ 

A few minor changes are suggested.

**Comment by vitodelgrec** [comment]
> Thanks for submitting these! ❤️
> 
> A few minor changes are suggested.

Thank you for the suggested changes @mfansler , all changes done and ready for review!

**Review by mfansler — APPROVED** [review]

**Pull request merged**